guides/v2.0/contributor-guide/contributing_docs.md

```diff
@@ -8,26 +8,42 @@ github_link: contributor-guide/contributing_docs.md
 
 Magento's team of technical writers reviews all issues and pull requests on a regular basis, and we do our best to address all issues as soon as possible. Working through the backlog takes time, though, so we appreciate your patience.
 
-<div class="bs-callout bs-callout-tip" id="tip" markdown="1">
+<div class="bs-callout bs-callout-info" id="info" markdown="1">
 If you are not sure where to start contributing, you can review our list of [suggested topics]({{ page.baseurl }}/contributor-guide/contributing_docs_suggested.html).
 </div>
 
-If you write and contribute a full topic, we will put your name (or company's name) at the top of the page, right under the title, and link it to your blog or website! We will also add your picture and a link to your Github account on the <a href="{{ page.baseurl }}/contributor-guide/quarterly-contributors.html#top-recent-contributors">Top recent contributors</a> page.
+If you write and contribute a full topic, we will put your name (or company's name) at the top of the page, right under the title, and link it to your blog or website! We will also add your picture and a link to your Github account on the [Top recent contributors]({{ page.baseurl }}/contributor-guide/quarterly-contributors.html#top-recent-contributors) page.
+
+<div class="bs-callout bs-callout-info" id="info" markdown="1">
+The 2.0 version of Magento DevDocs has reached its end of life (EOL). The doc set will still be available and visible on the Magento DevDocs site for you to access and reference. [Magento 2.0.18](https://devdocs.magento.com/guides/v2.0/release-notes/ReleaseNotes2.0.18CE.html) is the final 2.0x release, and as of March 2018, Magento 2.0x will no longer receive security patches, quality fixes, or documentation updates.
+</div>
+
+## Contribution guideslines
+We use [Markdown](http://daringfireball.net/projects/markdown/){:target="_blank"} to write our documentation, which is a simple markup language that we convert to HTML using [Kramdown](http://kramdown.gettalong.org/syntax.html){:target="_blank"}.
 
-## Contribution guidelines
-We use [Markdown](http://daringfireball.net/projects/markdown/){:target="\_blank"} to write our documentation, which is a simple markup language that we convert to HTML using [Kramdown](http://kramdown.gettalong.org/syntax.html){:target="\_blank"}. We have a <a href="{{ page.baseurl }}/contributor-guide/templates/basic_template.html" target="\_blank">template</a> to make getting started easy.
+You can update or add content to existing topics. We have all topics in versioned directories for 2.0, 2.1, 2.2, and onward.
```

Comment:
"You can update, or add content to, existing topics in their respective versioned directories (for 2.0, 2.1, 2.2, and onward)." Maybe?

Reply:
I dig it.